app/assets/javascripts/spotlight/clear_form_button.js

```diff
@@ -0,0 +1,21 @@
+Spotlight.onLoad(function() {
+  $('.btn-reset').ClearFormButton();
+});
+
+(function($) {
+  $.fn.ClearFormButton = function() {
+    var clearBtn = this;
+    var input = $(clearBtn).parent().prev('input');
+    $(input).on('keyup', function() {
```

Comment:
I wouldn't mind getting somebody a little more facile w/ jQuery to weigh in here, but a couple of things come to mind when looking over this.

1. Is `input` already a jQuery object at this point? (e.g. do we need to re-cast it as a jQuery object by calling `$(input)`).
1. Instead of casting `clearBtn` to a jquery object in a few places, can we do it when we assign the `clearBtn` var?
1. On the line below, can/should we use `this` which I _believe_ would be the input  in the `keypu` event scope?

One pattern I've seen (although admittedly haven't been good at using) is naming variables with a `$` when they are jQuery objects, just to know when we're dealing with one (e.g. `var $clearBtn = this`)

(FWIW, in a quick test drive of this branch I'm not getting the clear button, haven't tracked down the cause though)

Reply:
Thanks, @jkeck. I was able to take care of #1 and #2 and added `$` to the jQuery object variable names.

The clear button wasn't showing up until after a user starts typing in the form. 😅 I modified the function so that it checks for an input value when the page is first loaded. Let me know what you think.